docs/main.js

```diff
@@ -0,0 +1,272 @@
+var fs = require('fs')
+
+let num = [26, 236, 238, 27];
+const dates = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 30];
+const statusLen = 8;
+var buckets = {};
+let day = 0;
+var cnt = 0;
+
+const goodFortunes = [
+    {
+      "event": "睡覺",
+      "description": "品質良好，精神煥發"
+    },
+    {
+      "event": "做家務",
+      "description": "整潔使人心情愉悅"
+    },
+    {
+      "event": "冥想",
+      "description": "平靜心靈，緩解焦慮"
+    },
+    {
+      "event": "攝影",
+      "description": "捕捉到美好瞬間"
+    },
+    {
+      "event": "喝咖啡",
+      "description": "精力充沛燃燒脂肪"
+    },
+    {
+      "event": "朋友聚會",
+      "description": "充滿歡笑和美好回憶"
+    },
+    {
+      "event": "體育鍛鍊",
+      "description": "能量滿滿，效果顯著"
+    },
+    {
+      "event": "出遊" ,
+      "description": "好天氣，好心情"
+    },
+    {
+      "event": "吃大餐",
+      "description": "聯絡感情"
+    },
+    {
+      "event": "逛書店",
+      "description": "新書上架，打折推銷"
+    },
+    {
+      "event": "學新技能",
+      "description": "快速上手"
+    },
+    {
+      "event": "唱歌",
+      "description": "被星探發掘"
+    },
+    {
+      "event": "上課",
+      "description": "整天不累，100% 消化"
+    },
+    {
+      "event": "洗澡",
+      "description": "重獲能量"
+    },
+    {
+      "event": "請教問題",
+      "description": "問題皆獲高人指點"
+    },
+    {
+      "event": "網購",
+      "description": "心儀商品皆促銷"
+    },
+    {
+      "event": "放假",
+      "description": "休息充電，明日再戰"
+    },
+    {
+      "event": "早睡",
+      "description": "好夢連連"
+    },
+    {
+      "event": "早起",
+      "description": "朝氣蓬勃，神采飛揚"
+    },
+    {
+      "event": "發文章",
+      "description": "瀏覽數暴增"
+    },
+    {
+      "event": "點外賣",
+      "description": "準時到達，新鮮好吃"
+    },
+    {
+      "event": "做善事",
+      "description": "積善成福"
+    },
+    {
+      "event": "散步",
+      "description": "空氣良好，放鬆身心"
+    }
+];
+
+const badFortunes =  [
+    {
+      "event": "體育鍛鍊",
+      "description": "不慎受傷"
+    },
+    {
+      "event": "攝影",
+      "description": "照片全消失"
+    },
+    {
+      "event": "出遊",
+      "description": "天氣不晴朗"
+    },
+    {
+      "event": "吃大餐",
+      "description": "被要求請客"
+    },
+    {
+      "event": "學新技能",
+      "description": "屢試不爽，始終不懂"
+    },
+    {
+      "event": "唱歌",
+      "description": "嗓子發炎"
+    },
+    {
+      "event": "洗澡",
+      "description": "水溫不穩"
+    },
+    {
+      "event": "請教問題",
+      "description": "疑難雜症，均無解答"
+    },
+    {
+      "event": "網購",
+      "description": "錯過促銷"
+    },
+    {
+      "event": "放假",
+      "description": "隔日工作量倍增"
+    },
+    {
+      "event": "晚睡",
+      "description": "失眠，明日精神渙散"
+    },
+    {
+      "event": "晚起",
+      "description": "整天都不順"
+    },
+    {
+      "event": "發文章",
+      "description": "搜索枯腸，不知所云"
+    },
+    {
+      "event": "點外賣",
+      "description": "路況壅塞，餐點冷掉"
+    },
+    {
+      "event": "喝咖啡",
+      "description": "晚上失眠"
+    },
+    {
+      "event": "散步",
+      "description": "被害蟲咬傷"
+    }
+];
+
```

Comment:
fetch `goodFortunes` and `badFortunes` from [here](https://github.com/LifeAdventurer/generators/blob/main/fortune_generator/json/fortune.json)